packages/contractkit/src/wrappers/TransactionBuilder.ts

```diff
@@ -0,0 +1,103 @@
+import { keccak256 } from 'ethereumjs-util'
```

Comment:
Move this outside wrappers. Take a look about the Metadabuilder that nam did fro identity. I think there's an identity folder for it. Maybe have a governance folder